--- api_client.mustache (openapi-generator v7.25.0)
+++ api_client.mustache (tremendous)
@@ -111,7 +111,8 @@
 {{/async}}
 {{/compatibleWithPythonLegacyApi}}
         # Set default User-Agent.
-        self.user_agent = '{{{httpUserAgent}}}{{^httpUserAgent}}OpenAPI-Generator/{{{packageVersion}}}/python{{/httpUserAgent}}'
+        {{! Edited by Tremendous, line 114 !}}
+        self.user_agent = f"Tremendous Python v{tremendous.__version__}"
         self.client_side_validation = configuration.client_side_validation
 
 {{#async}}
